src/main/java/play/dev/filewatch/OS.java

```diff
@@ -9,9 +9,7 @@ enum OS {
   Other;
 
   static OS getCurrent() {
-    String osName = System.getProperty("os.name");
-    if (osName == null) return Other;
-    osName = osName.toLowerCase(Locale.ENGLISH);
+    final var osName = System.getProperty("os.name", "unknown").toLowerCase(Locale.ENGLISH);
```

Comment:
I moved this into one line, by using "unknown" if the os would be null, I think the code is a bit nicer to read, so you don't have to check for null and we only return `other` once below.